### PR TITLE
[4.0.0] Update MI doc on hashicorp environment variable support

### PR DESCRIPTION
**Pull request by Vathsan**
## Purpose
This PR updates the doc on environment variable support in HashCorp lookup.
Related PR: https://github.com/wso2-support/micro-integrator/pull/279

### CODE REVIEW

File: en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md [hunk 1]
```diff
@@ -183,6 +183,19 @@ Given below is a sample synapse property that uses a HashiCorp secret:
 <property name="HashiCorpSecret" expression="hashicorp:vault-lookup('path-name', 'field-name') />
 ```
 
+!!! Note
+    Support to access HashiCorp secrets using environment variables in synapse configurations is available as an update. To use this feature update your product version to 4.0.0.19 or higher. You may refer to [Updating WSO2 Micro Integrator]({{base_path}}/install-and-setup/setup/mi-setup/updating-mi) to get the latest updates.
```
Comment (samgnaniah) [inline en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md:187]: We need to mention the date of the update here. Let's say something like

"The capability to access HashiCorp secrets using environment variables in synapse configurations is added as an update on xx.xx.xxxx."

File: en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md [hunk 2]
```diff
@@ -183,6 +183,19 @@ Given below is a sample synapse property that uses a HashiCorp secret:
...
+    
+## Accessing HashiCorp secrets using environment variables in synapse configurations
+
+You can point to the secret stored in the HashiCorp vault from your synapse configurations using environment variables.
+
+Given below is a sample synapse property to retrieve HashiCorp secret using environment variables,
```
Comment (samgnaniah) [inline en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md:193]:
```suggestion
Given below is a sample synapse property to retrieve a HashiCorp secret using environment variables.
```

File: en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md (hunk 1 above)
Comment (Vathsan) [reply en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md:187]: Yes, we can add the date. But as I checked the new update tool does not provide any info on the update release date upon updating the product (we had the timestamp in WUM though). The new UMT update only contains the update level. IMO mentioning the update level will be more beneficial to the users. Also, MI-4.0.0 does not have WUM. WDYT?

Comment (samgnaniah) [reply en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md:187]: Ok. +1 to exclude the date if so.

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**